client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.js

```diff
@@ -63,9 +63,10 @@ const DataFilesProjectEditDescriptionModal = () => {
 
   const validationSchema = Yup.object().shape({
     title: Yup.string()
-      .min(1)
+      .min(3)
+      .max(70)
       .required('Please enter a title.'),
-    description: Yup.string()
+    description: Yup.string().max(800)
```

Comment:
Currently the validation message has both `title` and `description` in lowercase. I think we should either change the `validationSchema` and Form items' string field `name`'s to uppercase values, or add a custom validation message to the yup schema.

Perhaps the simplest would be a change to the fields, like so
```suggestion
    Description: Yup.string().max(800)
```

If it doesn't like that, we can add the custom validation message with a capitalized parameter name

Reply:
@rstijerina I don't think liked upcasing the field values. I manually gave it values.

Reply:
@duckonomy Sounds good. Can you add the same for `title` min values?

Reply:
Oops I'll have to fix the test.

Reply:
@rstijerina Got it in 🙂